consts.go

```diff
@@ -0,0 +1,6 @@
+package prismgoclient
+
+const (
+	// NtnxAPIKeyHeaderKey is the header key used for API key authentication
+	NtnxAPIKeyHeaderKey = "X-ntnx-api-key"
+)
```

Comment:
there's not need for this to be an exported part of the API. Keep it package local in internal and v4.